operations/helm/charts/mimir-distributed/templates/lib/securityContext.tpl

```diff
@@ -0,0 +1,12 @@
+{{/*
+Mimir common podSecurityContext
+Params:
+  ctx = . context
+  component = name of the component
+*/}}
+{{- define "mimir.lib.podSecurityContext" -}}
+{{- $componentSection := include "mimir.componentSectionFromName" . | fromYaml }}
+{{- /* Some components have security context in the values.yalm as podSecurityContext and others as securityContext */}}
+{{- $podSecurityContextSection := $componentSection.securityContext | default $componentSection.podSecurityContext -}}
```

Comment:
This doesn't work for the Mimir caches. Their podSecurityContext's are in the `memcached` values section, whereas this is looking for them in e.g. `index-cache` section.

Reply:
fixed